templates/components/common/responsive-img.html

```diff
@@ -43,6 +43,8 @@
 {{#unless lazyload '==' 'disabled'}}data-{{/unless}}srcset={{!--
     Generate the srcset string using all of the srcset_size_ properties in config.json. This will allow the browser to
     pick the best-sized image based on its own understanding of how it will render the page.
---}}"{{~#each theme_settings}}{{#if (occurrences @key "srcset_size_") ">=" 1}}{{getImage ../../image @key}} {{first (split this "x")}}w, {{/if}}{{/each}}"{{else if default_image}}src="{{cdn default_image}}"{{/if}}
+--}}"{{~#each theme_settings.srcset_sizes}}{{getImage ../../image @key}} {{first (split this "x")}}w{{#unless @last}}, {{/unless}}{{/each}}"
```

Comment:
this is the first time I've seen `{{~#` whats the `~` do?

Reply:
It trims whitespace: https://stackoverflow.com/questions/15953293/handlebars-whitespace-control